keyboards/viktus/minne_topre/rules.mk

```diff
@@ -0,0 +1,2 @@
+CUSTOM_MATRIX = lite
+SRC += analog.c ec.c
```

Comment:
```suggestion
QUANTUM_LIB_SRC += analog.c 
SRC += ec.c
```

Reply:
This change causes a ton of errors for info.JSON, and reverting back to what I had fixes them all.

Reply:
That's ... odd. It shouldn't cause any issues for the info.json.  but it may be worth compiling with `-c`, as that will clean out the build folder first, and help prevent weird issues.

But the reason for this, is that optimization and such can cause weird issues with some of the more hardware direct code, like this, and this helps prevent that from happening.

Reply:
Okay, Adding `-c` did fix that compile issue. Not sure why it did it either but at least it was an easy fix.